package.json

```diff
@@ -36,94 +36,95 @@
   },
   "dependencies": {
```

Comment:
No problem. I am reverting all `dependencies` except `yaml-js` as the old version doesn't seem to be compatible with webpack 2.